src/App.js

```diff
@@ -21,7 +27,7 @@ class App extends React.Component {
       headers: new Headers({
         'Content-Type': 'application/x-www-form-urlencoded',
       }),
-      body: "input="+ this.state.input
+      body: "input="+ this.state.input + "&in_lang=" + this.state.inputLanguage + "&out_lang" + this.state.outputLangauge
```

Comment:
Building up the string ourselves is error prone, could be easier to use https://stackoverflow.com/a/49701878.

Reply:
Would also be nice to do some validation, for example only sending when both in and out languages are set.